.typos.toml

```diff
@@ -18,6 +19,7 @@ extend-ignore-re = [
     "'bs_BA'",
     "<!\\[CDATA\\[non-classe\\]\\]>",
     "olt-manager\\.php",
+	"(?Rm)^.*//\\s*spellchecker:disable-line$", # Disable spell check on a single line with `// spellchecker:disable-line`.
```

Comment:
Oh noo!
This is not a good idea.
There many better ways to ignore false positive!

Reply:
Adding
```toml
"_anc_id" = "_anc_id"
```
to identifiers.